packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue

```diff
@@ -803,7 +803,7 @@ onMounted(async () => {
 	const modelConfigurationId = props.node.inputs[0].value?.[0];
 	if (!modelConfigurationId) return;
 	const modelConfiguration = await getModelConfigurationById(modelConfigurationId);
-	configuredInputModel = await getAsConfiguredModel(modelConfiguration);
+	configuredInputModel = await getAsConfiguredModel(modelConfiguration.id as string);
```

Comment:
No need to update, but I prefer using the proper type when available, as if we change it to something else, those typing bug will show up right away in the ts linter.

```suggestion
	configuredInputModel = await getAsConfiguredModel(modelConfiguration.id as ModelConfiguration["id"]);
```

Reply:
ModelConfiguration["id"] is type `string | undefined`
I see no issue using this and updating `getAsConfiguredModel` to the undefined scenario 
Instead here I am not doing this check as we know that it is defined at this point (or there are more serious issues)